File: v2/csharp/src/main/csharp/io/swagger/Model/Advisors.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace io.swagger.Model {
  public class Advisors {
    

    
    public List<Advisor> Advisors { get; set; }

    

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class Advisors {\n");
      
      sb.Append("  Advisors: ").Append(Advisors).Append("\n");
      
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  
  
}

File: v2/android/src/main/java/io/swagger/client/model/Advisors.java
package io.swagger.client.model;

import io.swagger.client.model.Advisor;
import java.util.*;

import com.wordnik.swagger.annotations.*;
import com.google.gson.annotations.SerializedName;


@ApiModel(description = "")
public class Advisors  {
  
  @SerializedName("advisors")
  private List<Advisor> advisors = new ArrayList<Advisor>() ;

  
  /**
   **/
  @ApiModelProperty(value = "")
  public List<Advisor> getAdvisors() {
    return advisors;
  }
  public void setAdvisors(List<Advisor> advisors) {
    this.advisors = advisors;
  }

  

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class Advisors {\n");
    
    sb.append("  advisors: ").append(advisors).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}
